web/src/features/map/Map.tsx

```diff
@@ -23,8 +23,8 @@ import {
   loadingMapAtom,
   mapMovingAtom,
   mousePositionAtom,
+  zoneIdAtom,
```

Comment:
Do we actually need an atom for tracking this - couldn't we just use the `useParams` hook from router instead? That way there's only one single source of truth, where here we now need to manually ensure these two are always in sync :)

Reply:
I like the params approach and there's currently an issue where the app will crash if the zoneId is invalid. Before I fix the crash we should decide on the approach. 

I went with a global state zoneId because of this comment ```// TODO: Selected feature-id should be stored in a global state instead (and as zoneId).``` 

and currently the routes are initialised in the LeftPanel, to have context of the zoneId in useParams() in map.tsx I'd need to initialize the routes above the map.

Let me know what you think!

Reply:
@madsnedergaard Let me know what you think about lifting the routes to the top level vs using an atom or another option.

Reply:
I think it makes sense to lift the routes to top level, it's already a bit confusing that it happens down in the panels only I think :)

Reply:
I can't remember how to do nested routing with React Router, but it should be doable :D